app/src/main/java/io/github/droidkaigi/confsched/fragment/MapFragment.java

```diff
@@ -170,7 +171,7 @@ private void renderMarkers(List<PlaceMap> placeMaps, GoogleMap googleMap) {
             MarkerOptions options = new MarkerOptions()
                     .position(new LatLng(placeMap.latitude, placeMap.longitude))
                     .title(getString(placeMap.nameRes))
-                    .icon(BitmapDescriptorFactory.fromResource(placeMap.markerRes))
+                    .icon(BitmapDescriptorFactory.fromBitmap(ResourceUtil.getBitmap(getContext(), placeMap.markerRes)))
```

Comment:
`BitmapDescriptorFactory.fromResource()` does not support vector drawable resource.
So I change resource to bitmap, and use `ResourceUtil.getBitmap()`. It works well.